sdk/postgresql/arm-postgresql-flexibleserver/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@azure/arm-postgresql-flexibleserver",
+  "author": "Microsoft Corporation",
+  "description": "PostgreSQLManagementFlexibleServerClient Library with typescript type definitions for node.js and browser.",
+  "version": "1.0.0",
```

Comment:
Are we sure this is the intended version to use for the upcoming release?

Reply:
I have created this PR for the `postgresql` team for their testing(purpose). We will decide on the version and packagename soon before merging this PR.

Reply:
Version Number is confirmed. Name is modified after discussion with Jeff.